client/src/API.js

```diff
@@ -0,0 +1,26 @@
+import axios from 'axios';
+
+export default {
+    getPokemonName: function () {
+        const min = 4;
+        const max = 11;
+        return axios.get(`https://uzby.com/api.php?min=${min}&max=${max}`);
+        // return axios.get('/api/pokemonName');
```

Comment:
comment out

Reply:
Comment out the min, max, and return statement?

Reply:
Oh wait jk nvm, I meant delete lol. But it's all good. Maybe just add a TODO in the current comment so we remember to add it in later.

Reply:
I just pushed the API changes to `test-server` so if you hit the heroku link with the same routes as the doc it should work